src/Containers/SelectBox.tsx

```diff
@@ -1,32 +1,34 @@
 import React from 'react';
 import styled from 'styled-components';
 
-export interface BoxProps {
+export interface SelectBoxProps extends React.HTMLAttributes<HTMLDivElement> {
     text?: string;
     align?: string;
-    selected?: boolean;
+    selected?: string;
```

Comment:
isn't this a boolean?

Reply:
isSelected if it is a boolean btw

Reply:
commit Formatting and prop type fixes that